compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRNode.java

```diff
@@ -125,6 +125,25 @@ public void accept(BIRVisitor visitor) {
         }
     }
 
+    /**
+     * A parameter.
+     *
+     * @since 0.995.0
+     */
+    public static class BIRParameter extends BIRNode {
+        public Name name;
+
+        public BIRParameter(DiagnosticPos pos, Name name) {
+            super(pos);
+            this.name = name;
+        }
+
+        @Override
+        public void accept(BIRVisitor visitor) {
+//            visitor.visit(this);
```

Comment:
This visit is commented out?

Reply:
yeah right now it's serialized without the help of the visitor, it only writes an integer, hence thought of writing that then and there, can do with the visitor as well. If the visitor is the better way, will add that  in the next PR